packages/visual-dev/src/components/RadioCard/RadioCard.tsx

```diff
@@ -67,45 +73,54 @@ const RadioGridDiv = styled.div`
   ${Styles.RadioGridStyle}
 `;
 
-export const RadioCardView = React.forwardRef<
-  React.ElementRef<"input">,
-  InputProps
->((props, forwardedRef) => {
-  const { value, optionValue, title, description, icon = "", ...rest } = props;
-
-  const selected = value === optionValue;
-
-  const icon_color = selected ? "var(--sq-action-primary-hovered)" : "";
-
-  return (
-    <RadioLabel htmlFor={rest.id} isChecked={selected}>
-      <RadioInput
-        {...rest}
-        type="radio"
-        checked={selected}
-        readOnly
-        ref={forwardedRef}
-      />
-      <LeftSegmentDiv isChecked={selected}>
-        {icon && <IconView icon={icon} size="40px" color={icon_color} />}
-      </LeftSegmentDiv>
-      <RightSegmentDiv>
-        <RadioTextDiv>
-          {title ? (
-            <div style={{ fontWeight: "bold", marginBottom: 4 }}>{title}</div>
-          ) : (
-            ""
-          )}
-          {description ? (
-            <div style={{ color: "var(--sq-text-subdued)" }}>{description}</div>
-          ) : (
-            ""
-          )}
-        </RadioTextDiv>
-      </RightSegmentDiv>
-    </RadioLabel>
-  );
-});
+const RadioCardView = React.forwardRef<React.ElementRef<"input">, InputProps>(
+  (props, forwardedRef) => {
+    const {
+      value,
+      optionValue,
+      title,
+      description,
+      customCSS,
+      icon = "",
+      ...rest
+    } = props;
+
+    const selected = value === optionValue;
+
+    const icon_color = selected ? "var(--sq-action-primary-hovered)" : "";
+
+    return (
+      <RadioLabel htmlFor={rest.id} isChecked={selected} customCSS={customCSS}>
+        <RadioInput
+          {...rest}
+          type="radio"
+          checked={selected}
+          readOnly
+          ref={forwardedRef}
+        />
+        <LeftSegmentDiv isChecked={selected}>
+          {icon && <IconView icon={icon} size="40px" color={icon_color} />}
+        </LeftSegmentDiv>
+        <RightSegmentDiv>
+          <RadioTextDiv>
+            {title ? (
+              <div style={{ fontWeight: "bold", marginBottom: 4 }}>{title}</div>
+            ) : (
+              ""
+            )}
+            {description ? (
+              <div style={{ color: "var(--sq-text-subdued)" }}>
+                {description}
+              </div>
+            ) : (
+              ""
+            )}
+          </RadioTextDiv>
```

Comment:
Can be cleaned up by switching ? to &&  
```suggestion
          <RadioTextDiv>
            {title && (
              <div style={{ fontWeight: "bold", marginBottom: 4 }}>{title}</div>
            ) }
            {description && (
              <div style={{ color: "var(--sq-text-subdued)" }}>
                {description}
              </div>
            )}
          </RadioTextDiv>
```

Reply:
Fixed in the bet-85 branch